src/rules/trailingCommaRule.ts

```diff
@@ -148,26 +249,66 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
         }
         const token = getChildOfKind(node, closeTokenKind, this.sourceFile);
         if (token !== undefined) {
-            return this.checkComma(list.hasTrailingComma, list, token.end);
+            return this.checkComma(list.hasTrailingComma, list, token.end, node);
         }
     }
 
-    private checkList(list: ts.NodeArray<ts.Node>, closeElementPos: number) {
+    private checkList(list: ts.NodeArray<ts.Node>, closeElementPos: number, originalNode: ts.Node) {
```

Comment:
related to comment above: change this parameter to `option: keyof CustomOptionValue`